common/com/twitter/intellij/pants/util/PantsUtil.java

```diff
@@ -421,6 +422,54 @@ public static GeneralCommandLine defaultCommandLine(@NotNull File pantsExecutabl
     }
   }
 
+  public static Collection<String> listMatchingTargets(@NotNull String projectPath, String selector) throws PantsException {
```

Comment:
I think we could use `com.twitter.intellij.pants.bsp.PantsTargetAddress` class here instead of `String` for selector 

(and it could be moved from bsp package)

Reply:
What is the conceptual difference between `com.twitter.intellij.pants.bsp.PantsTargetAddress` and `com.twitter.intellij.pants.model.PantsTargetAddress`?